composer.json

```diff
@@ -74,6 +74,7 @@
     },
     "require-dev": {
         "aws/aws-sdk-php": "^3.0",
+        "cakephp/chronos": "^1.1",
```

Comment:
Why is this a dev requirement if it is being used by the Carbon support class?

Reply:
For the tests

Reply:
So what happens when someone doesn't install chronos and then calls Carbon::immutable()?

Reply:
This won't work as (see PR description). Not everyone wants to use immutable `cakephp/chronos` and immutable datetimes. The choice is give to the developer, so the reason why this is is set in `suggest`